src/transformers/trainer_utils.py

```diff
@@ -2,12 +2,15 @@
 from typing import Any, Dict, List, NamedTuple, Optional, Union
 
 import numpy as np
-import torch
 
 from .file_utils import is_tf_available, is_torch_available
 from .tokenization_utils_base import ExplicitEnum
 
 
+if is_torch_available():
+    import torch
+
+
```

Comment:
@sgugger this isn't the *cleanest* as it isn't on parity with TensorFlow. I think this is acceptable until this file is refactored into two different files though?

Reply:
Yes, perfectly acceptable for now.